src/api/v0.rs

```diff
@@ -496,14 +476,13 @@ pub fn projects_info_default(projects: &str, db: State<DBPool>) -> CORS<JSON<Pro
 /// }
 /// ```
 ///
-pub fn projects_info(projects: &str, db: State<DBPool>, offset: i64, limit: i64) -> CORS<JSON<ProjectsInfoResponse>> {
-    info!("/projects/info/"; "projects" => projects, "offset" => offset, "limit" => limit);
+#[get("/projects/info/<projects>")]
+pub fn projects_info(projects: &str, db: State<DBPool>) -> CORS<JSON<ProjectsInfoResponse>> {
+    info!("/projects/info/"; "projects" => projects);
     let projects: Vec<&str> = projects.split(",").collect();
-    let result = get_projects_details(&db.conn(), &projects, offset, limit);
+    let result = get_projects_details(&db.conn(), &projects, 0, i64::max_value());
```

Comment:
I don't know if this is in a later patch, but you might as well just remove the last two arguments from get_projects_details.  We are also using it in src/bin/depclose.hs for the moment, but also with garbage arguments.

Also note - passing -1 as the limit seems to get all the results too.